training/cogvideox_image_to_video_lora.py

```diff
@@ -828,6 +833,10 @@ def load_model_hook(models, input_dir):
                     gradient_norm_before_clip = get_gradient_norm(transformer.parameters())
                     accelerator.clip_grad_norm_(transformer.parameters(), args.max_grad_norm)
                     gradient_norm_after_clip = get_gradient_norm(transformer.parameters())
+                    logs.update({
```

Comment:
Should we not consider if we're using DeepSpeed and log it accordingly?

I referring to the following comment:

```
# gradnorm + deepspeed: https://github.com/microsoft/DeepSpeed/issues/4555
```

Reply:
we cannot calculate grad_norm for deepspeed, so this will be None